### PR TITLE
Add Composio to the Hacktoberfest Swag List

### PR DESCRIPTION
**Pull request by Brijeshthummar02**
issue #916 

1. [x] I have read the [Contributing.md](https://github.com/crweiner/hacktoberfest-swag-list/blob/master/docs/contributing.md) file and formatted this PR correctly
2. [x] I'm not adding a company from the blocklist
3. [x] I make sure to fix things promptly if an error or suggestion comes up

Thanks and Happy Hacktoberfest! :tada:
Tagging @crweiner to take a look. :eyes:

### CODE REVIEW

File: docs/contributing.md [hunk 1]
```diff
@@ -50,6 +50,16 @@ Find the first letter of your company within the list, then add your information
...
 ```
 
+### C
+
+#### **Composio**
+
+- **Swag**: Stickers, T-shirt
+- **Requirements**: Make 4 good PRs against 'hacktoberfest' topics in any of the Composio repositories.
+- **How to sign up**: Register for Hacktoberfest [here](https://hacktoberfest.com/).
+- **Issues**: Optional link to Hacktoberfest tagged issues in Composio repositories.
```
Comment (vimode) [inline docs/contributing.md:60]:
```suggestion
- **Issues**: Open issues [tagged as "hacktoberfest"](https://github.com/ComposioHQ/composio/issues?q=sort%3Aupdated-desc+is%3Aissue+is%3Aopen+label%3Ahacktoberfest)
```

File: docs/contributing.md [hunk 2]
```diff
@@ -50,6 +50,16 @@ Find the first letter of your company within the list, then add your information
...
+- **Notes**: Ensure to submit PRs before October 31, 2024, for swag eligibility.
```
Comment (vimode) [inline docs/contributing.md:61]:
```suggestion
- **Notes**: [Check out their announcement post for more information.](https://github.com/ComposioHQ/composio/blob/master/HACKTOBER_FEST_2024.md)
```

File: docs/contributing.md [hunk 3]
```diff
@@ -50,6 +50,16 @@ Find the first letter of your company within the list, then add your information
 ### A
 ```
 
+### C
+
+#### **Composio**
+
+- **Swag**: Stickers, T-shirt
```
Comment (vimode) [inline docs/contributing.md:57]:
```suggestion
- **Swag**: Stickers and T-shirt
```

**Review by vimode — CHANGES_REQUESTED** [review]
These would help contributors to reach the necessary links in minimal clicks!

**Comment by crweiner** [comment]
Hi @Brijeshthummar02, thank you for your PR, and also thank you @vimode for suggesting some good corrections. However, this PR is against the Contributing.md file, not the actual list (List.md).

There is another existing PR open for Composio, PR #917, that is in the correct file, so I will move forward with that other one. @Brijeshthummar02, I encourage you to try again with a different PR to add someone else to the list and be extra careful!

**Pull request closed without merging**